wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift

```diff
@@ -103,6 +97,13 @@ public enum SemanticColors {
 
         // Audio Icon
         static let foregroundAudio = UIColor(light: Asset.Colors.black, dark: Asset.Colors.white)
+
+        // System Message Icon Colors
+        static let foregroundExclamationMarkInSystemMessage = UIColor(light: Asset.Colors.red500Light, dark: Asset.Colors.red500Dark)
+        static let foregroundCheckMarkInSystemMessage = UIColor(light: Asset.Colors.green500Light, dark: Asset.Colors.green500Dark)
```

Comment:
Do we use `foregroundExclamationMarkInSystemMessage` and `foregroundCheckMarkInSystemMessage` ?

Reply:
Yes!